conf/app.routes

```diff
@@ -62,7 +62,7 @@ GET        /shuttering/:shutterType/:env                uk.gov.hmrc.cataloguefro
 GET        /shutter-events                              uk.gov.hmrc.cataloguefrontend.shuttering.ShutterEventsController.shutterEvents
 GET        /shutter-events/list                         uk.gov.hmrc.cataloguefrontend.shuttering.ShutterEventsController.shutterEventsList(env: Environment, serviceName: Option[String] ?= None)
 
-GET        /whats-running-where-heritage                uk.gov.hmrc.cataloguefrontend.whatsrunningwhere.WhatsRunningWhereController.heritageReleases
-GET        /whats-running-where                        uk.gov.hmrc.cataloguefrontend.whatsrunningwhere.WhatsRunningWhereController.ecsReleases
+GET        /whats-running-where-heritage                uk.gov.hmrc.cataloguefrontend.whatsrunningwhere.WhatsRunningWhereController.heritageReleases(showDiff: Boolean ?= false)
```

Comment:
How will this feature be discovered if the query param has a default value? The WRW page can have a button to toggle this flag.

Reply:
Feature, not a bug! - it's in order to stealth release to get feedback before making discoverable